locations/wuhlipunks.yml

```diff
@@ -9,7 +9,6 @@ height: 5
 community: true
 
 hosts:
-
```

Comment:
Diese Leerzeilen sind nicht erlaubt? Finde die manchmal ganz angenehm für die Übersichtlichkeit. (Ist in den locations natürlich jedem selbst überlassen.)